### ISSUE TITLE
[#9] 2/2: SoundSource + RodioSoundPlayer実装

### ISSUE THREAD

**Issue by takemo101** [takemo101/agent-bench#32]
## 概要

サウンドソースの検出機能とrodio v0.21を使用したサウンドプレイヤーを実装する。

## 親Issue

- Epic: #9

## 推定規模

| 項目 | 値 |
|------|-----|
| コード量 | ~130行 |
| ファイル数 | 2件 |

## 対象ファイル

- `src/sound/source.rs` (新規)
- `src/sound/player.rs` (新規)
- `src/lib.rs` (更新 - モジュール追加)

### src/sound/source.rs
- [ ] `SoundSource` enum定義
  ```rust
  #[derive(Debug, Clone, PartialEq, Eq)]
  pub enum SoundSource {
      System { name: String, path: PathBuf },
      Embedded { name: String },
  }
  ```
- [ ] `discover_system_sounds() -> Vec<SoundSource>`
  - `/System/Library/Sounds/` を走査
  - 対応フォーマット: .aiff, .wav, .mp3
- [ ] `get_default_source() -> SoundSource`
  - Glass.aiff → Ping.aiff → 埋め込み の優先順

### src/sound/player.rs
- [ ] `RodioSoundPlayer` struct定義
  ```rust
  pub struct RodioSoundPlayer {
      _stream: OutputStream,
      stream_handle: OutputStreamHandle,
      disabled: bool,
  }
  ```
- [ ] `new(disabled: bool)` - ストリーム初期化
  - `OutputStreamBuilder::open_default_stream()` 使用
- [ ] `SoundPlayer` トレイト実装
  - `play(source)` - 非同期再生
  - `is_available()` - デバイス利用可能チェック
- [ ] サウンド再生ロジック
  - `Decoder::try_from()` でデコード
  - `Sink::connect_new()` で再生
  - `sink.detach()` でバックグラウンド再生

### src/lib.rs
- [ ] `pub mod sound;` 追加

### テスト実装
- [ ] `discover_system_sounds` 単体テスト
- [ ] `disabled=true` でスキップ確認
- [ ] モックプレイヤーテスト
- [ ] ファイル未検出時のフォールバックテスト

## 技術的ポイント

- rodio v0.21 新API (`OutputStreamBuilder`, `Sink::connect_new`)
- `sink.detach()` による非ブロッキング再生
- システムサウンド未検出時は埋め込みサウンドにフォールバック

## 完了条件

- [ ] 実装完了（130行以下）
- [ ] `cargo build` 成功
- [ ] `cargo test` パス
- [ ] `cargo clippy` 警告なし
- [ ] サウンド再生確認（手動）
- [ ] PR作成・レビュー

## 依存

- #31（基盤実装完了後に着手）

## 参照設計書

- [サウンド再生詳細設計書](docs/designs/detailed/pomodoro-timer/sound-playback.md) セクション 3, 4, 7, 8

---
🤖 このIssueは `/decompose-issue 9` により自動生成されました
